index.html

```diff
@@ -72,9 +129,258 @@ <h2 class="greeting">
                         </li>
                   </ul>
                 </div> 
+>>>>>>> main
   
         </section>
-    </header>
-  </div>
+</header>
+      <!----------------------------------------End of Header section--------------------------- -->
+      <!-- --------------------------------------Start of Main and Work section 1--------------------------------->
+      <main>
+      <div class="work-section">
```

Comment:
- [ ] [REQUIRED] Please implement the correct font size to your `headings` and give the correct style to your `button`🙏🏼

![image](https://user-images.githubusercontent.com/63915024/228899105-81e11ad0-7d1a-40ae-9378-9ab9e0f25008.png)